controllers/nf/free5gc_smf/resources.go

```diff
@@ -139,50 +133,52 @@ func createDeployment(log logr.Logger, configMapVersion string, upfDeployment *n
 	return deployment, nil
 }
 
-func createConfigMap(log logr.Logger, upfDeployment *nephiov1alpha1.UPFDeployment) (*apiv1.ConfigMap, error) {
-	namespace := upfDeployment.Namespace
-	instanceName := upfDeployment.Name
+func createService(nfDeployment *nephiov1alpha1.NFDeployment) *apiv1.Service {
+	namespace := nfDeployment.Namespace
+	instanceName := nfDeployment.Name
 
-	n4ip, err := controllers.GetFirstInterfaceConfigIPv4(upfDeployment.Spec.Interfaces, "n4")
-	if err != nil {
-		log.Error(err, "Interface N4 not found in UPFDeployment Spec")
-		return nil, err
+	labels := map[string]string{
+		"name": instanceName,
 	}
 
-	n3ip, err := controllers.GetFirstInterfaceConfigIPv4(upfDeployment.Spec.Interfaces, "n3")
-	if err != nil {
-		log.Error(err, "Interface N3 not found in UPFDeployment Spec")
-		return nil, err
+	service := &apiv1.Service{
```

Comment:
Interesting... how was this removed earlier? I still see this code in the main branch?